extensions/reclaim-ai/package.json

```
@@ -11,7 +11,8 @@
     "clayton_knapp",
     "scottfseely",
     "sandygifford",
-    "dmkato"
+    "dmkato",
+    "ridemountainpig"
```

```suggestion
    "ridemountainpig"
```

Thanks for your contribution! Please remove this as we prefer to have only current Reclaim.ai employees and contractors as CODEOWNERS, however we will be happy to add you to the list of contributors here:

https://github.com/raycast/extensions/tree/a4031c2ade3a42c5ceb36234f6456977baf26e52/extensions/reclaim-ai#why-arent-these-contributors-listed-in-the-packagejson-contributors-field

---

We can add other contributors to `pastContributors`, they will not get pinged but still credited